tests/validation_tests/specific_validation_tests/pathfinders/test_round_1.py

```diff
@@ -29,11 +29,15 @@ def mock_control_mappings():
         "programme_id_to_allowed_bespoke_outcomes": {"PF-BOL": []},
         "intervention_theme_to_standard_outputs": {
             "Improving the quality of life of residents": ["Amount of existing parks/greenspace/outdoor improved"],
+            "Enhancing sub-regional and regional connectivity": [
```

Comment:
Is this the "incorrect mapping"? Should we just add a note here to state that so it's easier to change later on?